### ISSUE TITLE
[Bug] Build stuck at "Building native binary with IL2CPP"

### ISSUE THREAD

**Issue by iouared** [Unity-Technologies/arfoundation-samples#785]
**Describe the bug**
Hello, i'm new to AR Foundation and i just tried to build samples of SimpleAR. I tried this with two different computer but the issue is the same: when i build to my android tablet, the building is stuck at "Building native binary with IL2CPP". I search a long time for different solutions like change computer, on change the PATH, i tried to rename folder to have ASCII character but i'm still stuck at this stage.
Thank you in advance for your help (and sorry for my bad english, i'm french)
[Editor.log](https://github.com/Unity-Technologies/arfoundation-samples/files/6244771/Editor.log)
[Editor.log](https://github.com/Unity-Technologies/arfoundation-samples/files/6244773/Editor.log)
![issueunity1](https://user-images.githubusercontent.com/23238164/113319881-31417580-9312-11eb-95ac-5294fefa9ede.PNG)

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy Tab A7
 - OS: Android Nougat
 - Unity 2020.3.2f1
 - ARFoundation version 4.0

**Comment by tdmowrer** [comment]
Are you able to build a vanilla Android project for Arm 32 and 64 with the IL2CPP backend enabled? That is, without any AR Foundation related packages?

**Comment by iouared** [comment]
Hello tdmowrer, thanks for your answer. Yes i already build android project without any AR Foundation related packages, this afternoon for exemple with a simple app.

**Comment by tdmowrer** [comment]
I tried building `main` of this repo for Android using Unity 2020.3.2f1 on both Mac and Windows without issue. Your Editor log doesn't reveal anything; have you tried to build on more than one machine?

**Comment by iouared** [comment]
I have two computer, one laptop and one desktop both on Windows 10. I tried two build with the two of them. I already reinstall unity several times. Each build i wait like 30/40 minutes, unity froze. On the task manager i can see that the proc and the RAM are well use and unity editor status is "no answer" and still stuck on "Building native binary with IL2CPP". When I build a new project with the AR template that unity 2020.3.2f1 offer, it works perfectly and i don't understand why; Maybe some packages related to ar foundation sample ?

**Comment by tdmowrer** [comment]
We are unable to reproduce this. Could you please log a bug by following the instructions at https://unity3d.com/unity/qa/bug-reporting and post the case ID here?

**Comment by iouared** [comment]
Hello, I tried for the past days to resolve these and here is my conclusion : 
I try to build the project with my friend's computer (a gamer config) and it succeed. Actually my computer at home were too old to build this in short time and it takes several hours (I never wait so long and my proc and my ram were so low that it feel like a bug). To optimize the building time I desactivated the "Baked Global Illumination" option in Window -> Rendering -> Lighting.
I'm sorry for this newbie mistake. I hope it may help some people in the futur and sorry for my bad English.
Thanks again for your time.

**Comment by tdmowrer** [comment]
That is a very long time to bake lighting, but I'm glad that you figured it out :)

**Comment by jiungerich** [comment]
I just found this same issue collaborating with another developer using 2020.3.12f1 and AR Foundation 4.1.7.  The project was working fine, then some very minor changes were made in the background color of the UI and an extra image was added to a panel.  On the other developer's windows machine (building for Android), build times went from 2min to hours.  It hangs in"Building native binary with IL2CPP" if "Build and Run" is clicked.  If just build is clicked, the build gets past that but then hangs in "stripping assemblies".  I loaded a git repo of the project to a newer mac pro and it compiles in 2-3min but is slow on the steps where it hangs on the windows machine.  It is a very simple SLAM app that just displays a couple of 2D artworks in AR and has 2 scripts.
